src/modules/filters/filters.repository.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+import { DataSource } from 'typeorm';
+
+@Injectable()
+export class FiltersRepository {
+  constructor(private readonly dataSource: DataSource) {}
+
+  async getAllFilters() {
+    const [brands, categories, colors, materials, sizes, styles] =
+      await Promise.all([
+        this.dataSource.query('SELECT * FROM brands'),
```

Comment:
why you use pure sql not typeorm functions?

Reply:
I used it because I didn't find a method how to use typeorm, but I refactored it.